.cirrus.yml

```diff
@@ -16,16 +17,19 @@ fedora_30_task:
     matrix:
       - SETUP: setup.py develop --user
       - SETUP: -m pip install .
+    matrix:
+      - VT_TYPE: qemu
+      - VT_TYPE: libvirt
   setup_script: &setup_scr
     - $PYTHON --version
     - test -z $AVOCADO_SRC || $PYTHON -m pip install $AVOCADO_SRC
     - $PYTHON $SETUP
   bootstrap_script: &bootstrap_scr
-    - $PYTHON -m avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
+    - $PYTHON -m avocado vt-bootstrap --vt-type=$VT_TYPE --vt-skip-verify-download-assets --yes-to-all
   list_script: &list_scr
-    - $PYTHON -m avocado list
+    - $PYTHON -m avocado list --vt-type=$VT_TYPE
   dry_run_script: &dry_run_scr
-    - $PYTHON -m avocado --show all run --dry-run -- boot
+    - $PYTHON -m avocado --show all run --vt-type=$VT_TYPE --dry-run -- boot
```

Comment:
With `tp-libvirt` added I'd suggest using `io-github-autotest-qemu.boot` instead of the `boot` test only as libvirt contains several `boot` tests (52 actually). Anyway it's not a strong request, just a suggestion...